pnpm-lock.yaml

```diff
@@ -1,9 +1,5 @@
 lockfileVersion: '6.0'
 
-settings:
-  autoInstallPeers: true
-  excludeLinksFromLockfile: false
-
```

Comment:
Unsure why there are changes in this file other than the addition of `pubsub-js` and `@types/pubusb-js`..

Reply:
I just ran the following command on my local machine

```sh
pnpm add pubsub-js
pnpm add pubsub-js -D
```